.travis.yml

```diff
@@ -11,3 +11,7 @@ branches:
 cache:
   directories:
     - node_modules
+env:
+  global:
+    - SAUCE_USERNAME=cerner
```

Comment:
I'm not quite sure what you are asking. This is defining SAUCE_USERNAME as an environment variable with respect to the Travis build.